grace_navigation/RangeBearing range_bearing
---
geometry_msgs/PoseStamped pose